### DISCUSSION TITLE
Would mydumper 10.3 ever set a global locking on MySQL 8.0.32?

### DISCUSSION THREAD

**Discussion by VMEMachuca** [mydumper/mydumper#1294] [Q&A]
We are using mydumper 0.10.3, built against MySQL 5.7.33-36 attempting to export from MySQL Server version reported as: 8.0.32-24.

This is what we see in the log:

** (mydumper:206074): WARNING **: 05:46:54.728: Using trx_consistency_only, binlog coordinates will not be accurate if you are writing to non transactional tables.

** (mydumper:206074): CRITICAL **: 11:50:36.592: Couldn't acquire global lock, snapshots will not be consistent: Query execution was interrupted

** (mydumper:206074): CRITICAL **: 11:50:36.635: Failed to connect to database: Can't connect to MySQL server on 'sql.xx.xx.xx.xxx.x.xx' (111)

**Comment by davidducos** [comment]
Hi @VMEMachuca, 
There are multi issues. Imposible to give you an exact diagnose. 
Do not use --trx-consistency-only.

FTWRL was not working, why don't you try with --lock-all-tables

About the last error, seems that the server crash, am I right?
